### ISSUE TITLE
Update JMSSerializerBundle dependency

### ISSUE THREAD

**Issue by gseidel** [Sylius/Sylius#3389]
On June 2015 the JMSSerializerBundle has released the stable version 1.0.0
It would be nice if we can update to this release
https://github.com/schmittjoh/JMSSerializerBundle/releases

**Comment by michalmarcinkowski** [comment]
We should definitely make an update. Thanks for rising that up :+1:

**Comment by coudenysj** [comment]
The symonfy-cmf/create-bundle doesn't like the upgrade:

```shell
$ composer require jms/serializer-bundle "~1.0"
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - symfony-cmf/create-bundle 1.2.2 requires jms/serializer-bundle ~0.12 -> no matching package found.
    - symfony-cmf/create-bundle 1.2.2 requires jms/serializer-bundle ~0.12 -> no matching package found.
    - symfony-cmf/create-bundle 1.2.1 requires jms/serializer-bundle ~0.12 -> no matching package found.
    - symfony-cmf/create-bundle 1.2.0 requires jms/serializer-bundle ~0.12 -> no matching package found.
    - Installation request for symfony-cmf/create-bundle ~1.2 -> satisfiable by symfony-cmf/create-bundle[1.2.0, 1.2.1, 1.2.2].

Potential causes:
 - A typo in the package name
 - The package is not available in a stable-enough version according to your minimum-stability setting
   see <https://groups.google.com/d/topic/composer-dev/_g3ASeIFlrc/discussion> for more details.

Read <https://getcomposer.org/doc/articles/troubleshooting.md> for further common problems.

Installation failed, reverting ./composer.json to its original content.
```

**Comment by michalmarcinkowski** [comment]
It's fixed on `master` https://github.com/symfony-cmf/CreateBundle/pull/142. Waiting for release then :wink:

**Comment by dbu** [comment]
i think we can release 1.3.0 of CreateBundle. @lsmith77 do you see any problem or can i just tag that? other bundles are not all ready, but the cmf bundle seems good.

**Comment by Danita** [comment]
Hi guys! First of all, thank you for your great work! I'm reporting this here because https://github.com/Sylius/SyliusRbacBundle doesn't have "Issues" enabled.

We're trying to use RbacBundle in our project, but we can't install it because it has a dependency on ResourceBundle, which also depends on JMSSerializerBundle at v0.13.0. We're using v1.1.0 of the latter and it's very difficult for us to rollback to v0.13.0, so there lies our conflict. 

Is there any chance you'll update ResourceBundle and RbacBundle to depend on JMSSerializerBundle v1.x in the near future? Thanks a lot! :smile:

**Comment by michalmarcinkowski** [comment]
We would like to update it ASAP, but we're dependent on the `symfony-cmf/create-bundle`. @dbu @lsmith77 when do you think we can expect the release? Is there something blocking you?

**Comment by lsmith77** [comment]
allowing newer JMS serializer bundle versions can be done in a patch release too

**Comment by lsmith77** [comment]
so maybe we should back port https://github.com/symfony-cmf/CreateBundle/commit/894bd1c25ebf054054453564013ec0831bc8db2d into the 1.2 branch imho ..

**Comment by michalmarcinkowski** [comment]
@lsmith77 sounds good to me :)

**Comment by dbu** [comment]
i am trying that backport. can somebody of you check if things work out fine with the branch of https://github.com/symfony-cmf/CreateBundle/pull/145 ?

**Comment by lsmith77** [comment]
FYI CreateBundle 1.2.3 has been released https://github.com/symfony-cmf/CreateBundle/releases/tag/1.2.3

**Comment by pjedrzejewski** [comment]
@lsmith77 Thanks Lukas!

**Comment by michalmarcinkowski** [comment]
Fixed in #3818
